### DISCUSSION TITLE
Embed Next.js build in another project

### DISCUSSION THREAD

**Discussion by safeerraees** [vercel/next.js#27614] [Help]
I have a situation where I have to embed a Next.js project in html in another project.  I have done this already with a Reactjs project with the single bundle js and css file bundler creates. But I don't know how can I achieve this with Next.js. 

React build folder:
![image](https://user-images.githubusercontent.com/45023860/127618795-1ea9c2dd-99fa-44fe-bf37-154efca89e12.png)

Next build folder:
![image](https://user-images.githubusercontent.com/45023860/127618997-9cd6c99c-3e40-45c8-bac1-49627112b145.png)

If the build can be converted to single bundle js and css file that would be great.

**Comment by richardnikolas** [comment]
@safeerraees bro, were you able to solve this problem? I'm having the exact same issue here...

**Comment by erikmellum** [comment]
Working on the same thing here: https://github.com/vercel/next.js/discussions/46646

I feel like if code splitting could be turned off this would work.
